### ISSUE TITLE
Comment Notify PHP error (TypeError about array_filter)

### ISSUE THREAD

**Issue by leeomara** [ytgov/yukon-ca#610]
Certain pages (e.g. editing a user) are causing a site crash that seems related to the Comment Notify module

On YG dev, visiting a user edit page results in a site crash with this message

> The website encountered an unexpected error. Try again later.

```
Got error 'PHP message: Uncaught PHP Exception TypeError: "array_filter(): 
Argument #1 ($array) must be of type array, null given" at 
.../docroot/modules/contrib/comment_notify/comment_notify.module line 35', 
referer: https://.../en/user/1
```

Maybe this is the same as this Drupal.org issue?
  https://www.drupal.org/project/comment_notify/issues/3392374

This module appears to have landed as part of #577.

**Comment by w3creatives** [comment]
We have uninstalled this module. Was it installed for #508 ?

**Comment by leeomara** [comment]
> Was it installed for #508 ?

Not #508. This was installed as part of #577. Particularly "Blog comment alert."

**Comment by lgontard** [comment]
@leeomara Can you review this?

**Comment by leeomara** [comment]
This particular error is resolved, however the workaround (uninstalling the module that caused the error) undoes the "Blog comment alert" feature (which landed in #577)
